ast/builtins.go

```diff
@@ -2604,7 +2604,10 @@ var GlobMatch = &Builtin{
 	Decl: types.NewFunction(
 		types.Args(
 			types.Named("pattern", types.S),
-			types.Named("delimiters", types.NewArray(nil, types.S)).Description("glob pattern delimiters, e.g. `[\".\", \":\"]`, defaults to `[\".\"]` if unset."),
+			types.Named("delimiters", types.NewAny(
```

Comment:
I think this line only gives a call by type `ArrayString` or `null`. So any types else have raised error before function.

Reply:
That's for static inputs. There are still inputs whose type is only known at runtime. For example `glob.match(input.m, input.d, input.x)`

Reply:
I have made a change to return `OperandError` with default.